firstvoices/backend/views/sites_views.py

```diff
@@ -79,7 +79,6 @@ def get_detail_queryset(self):
                 "menu",
                 "language",
                 "homepage",
-                "homepage",
```

Comment:
Ha! Thanks for catching this.

I'm looking at the rest of this file and I realized that other_sites in the list method is missing prefetching. Could you add it there as well (it can match the prefetching from the Language query, just adjusted to work on a Site queryset.

Reply:
Good call. I have added prefetching for other_sites to this PR.